src/pages/SearchPage.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {movies} from '../mocks/movie';
+import {SearchHeader} from '../components/SearchHeader';
+import {ResultHeader} from '../components/ResultHeader';
+import {MovieList} from '../components/MovieList';
+import {Footer} from '../components/Footer';
+
+export class SearchPage extends React.PureComponent {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      sortBy: 'release_date'
+    }
+  }
+
+  onSortChange = (sortBy) => {
```

Comment:
One common convention that is used in React world is to name this handlers for events starting with handle, then in your case could be `handleSortChange`